core/src/avm2/globals/xml.rs

```diff
@@ -10,6 +11,16 @@ use crate::avm2::string::AvmString;
 use crate::avm2::{Activation, Error, Multiname, Object, Value};
 use crate::avm2_stub_method;
 
+fn ill_formed_markup_err<'gc>(activation: &mut Activation<'_, 'gc>) -> Value<'gc> {
```

Comment:
btw, AFAIK the usual pattern we do here is: return the Result, then in the caller do `return Err(AvmError(ill_formed_markup_err(...)?));` instead of a panic.